README.md

```diff
@@ -1,57 +1,177 @@
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[User will have access to all the projects and used technologies]**
+- **[User will be updated with the recent working updates]**
+- **[User will be able to contact and purchase merchandise]**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+By following below steps a new developer can make instant use of the project.
+
+Get a local copy up and running now:
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Abdel-Khafid SALAOU**
+
+- GitHub: [@githubhandle](https://github.com/adeola003)
+- Twitter: [@twitterhandle](https://twitter.com/khadaf6)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [Place registration form and merchandise basket] **[Upgrade the hmtl file]**
+- [Total of three projects on discussions] **[Create a description of main and side projects]**
+- [Include pre-Covid working experience] **[Update on latest working experiences]**
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to come in touch with the author and arrange a meeting and make your contributions truly valuable. 
+
+## ⭐️ Show your support <a name="support"></a>
+
+As this is one of the first projects of the author, please be kind enough and show your support by DMing on twitter or by contributing to this project. All your efforts are highly appreciated and welcomed!
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+For this project many hours have been invested. For that reason, I would like to thank all my family that supported me in this long journey, the microverse team and my partners.
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Who will benefit from such a tool at the end of the day?]**
+
+  - [New Micronauts]
+
+- **[From where do I begin my journey with your Company?]**
+
+  - [It all starts with the filling of our registration form]
+
+  ## 📝 License <a name="license"></a>
+
+  This project is [MIT] licensed.

```

Comment:
- **[OPTIONAL]** Please 🙏🏽 make sure that the MIT link works as expected. We could correct this by using 👉🏽  `[MIT](license.md)` for a professional README file.

Reply:
Thank you very much.